keyboards/k_type/is31fl3733.c

```diff
@@ -0,0 +1,236 @@
+/* Copyright 2017 Jason Williams
```

Comment:
Similarly I think these two could do with a `-dual` suffix or something, to distinguish it from the core version.